tests/new_api/test_device.py

```diff
@@ -383,26 +391,38 @@ def test_preprocess_state_prep_middle_op_decomposition(self, op, decomp_depth):
     @pytest.mark.parametrize("theta, phi", list(zip(THETA, PHI)))
     @pytest.mark.parametrize(
         "mp",
-        [
-            qml.probs(wires=[1, 2]),
-            qml.probs(op=qml.Z(2)),
-            qml.expval(qml.Z(2)),
-            qml.var(qml.X(2)),
-            qml.expval(qml.X(0) + qml.Z(0)),
-            qml.expval(qml.Hamiltonian([-0.5, 1.5], [qml.Y(1), qml.X(1)])),
-            qml.expval(2.5 * qml.Z(0)),
-            qml.expval(qml.Z(0) @ qml.X(1)),
-            qml.expval(qml.operation.Tensor(qml.Z(0), qml.X(1))),
-            qml.expval(
-                qml.SparseHamiltonian(
-                    qml.Hamiltonian([-1.0, 1.5], [qml.Z(1), qml.X(1)]).sparse_matrix(
-                        wire_order=[0, 1, 2]
-                    ),
-                    wires=[0, 1, 2],
-                )
-            ),
-            qml.expval(qml.Projector([1], wires=2)),
-        ],
+        (
+            [
+                qml.probs(wires=[1, 2]),
+                qml.probs(op=qml.Z(2)),
+                qml.expval(qml.Z(2)),
+                qml.var(qml.X(2)),
+                qml.expval(qml.X(0) + qml.Z(0)),
+                qml.expval(qml.Hamiltonian([-0.5, 1.5], [qml.Y(1), qml.X(1)])),
+                qml.expval(2.5 * qml.Z(0)),
+                qml.expval(qml.Z(0) @ qml.X(1)),
+                qml.expval(qml.operation.Tensor(qml.Z(0), qml.X(1))),
+                qml.expval(
+                    qml.SparseHamiltonian(
+                        qml.Hamiltonian([-1.0, 1.5], [qml.Z(1), qml.X(1)]).sparse_matrix(
+                            wire_order=[0, 1, 2]
+                        ),
+                        wires=[0, 1, 2],
+                    )
+                ),
+                qml.expval(qml.Projector([1], wires=2)),
+            ]
+            if device_name != "lightning.tensor"
```

Comment:
Could we instead use `pytest.skip` in the test and filter unsupported operations?